eng/pipelines/templates/Initialize_Build.yml

```diff
@@ -4,8 +4,7 @@ steps:
   inputs:
     scriptType: "inlineScript"
     inlineScript: |
-      . $(Build.Repository.LocalPath)\\scripts\\utils\\PostGitCommitStatus.ps1
-      CheckVstsPersonalAccessToken -VstsPersonalAccessToken $(System.AccessToken)
+       CheckVstsPersonalAccessToken -VstsPersonalAccessToken $(System.AccessToken)
```

Comment:
System.AccessToken always has access.  This Vsts PAT check was added in a time when we were using a different PAT.  This can be deleted as well.